packages/patternfly-4/react-integration/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
```

Comment:
Is it possible to reference a tsconfig in `react-core`? Trying to minimize duplicate configuration.

Reply:
I'll check.

Reply:
tsconfig in react core is slightly different.  I removed the one that was in cypress directory when I added this one.